### ISSUE TITLE
Error message updates for Name field (A, AAAA, MX, CNAME, TXT)

### ISSUE THREAD

**Issue by gabo0oo** [cisagov/getgov#5117]
### Issue description

In our recent design & accessibility review for II Beam (#4727 | [Slack Canvas](https://dhscisa.enterprise.slack.com/docs/T02QH7E1MHA/F097U65KVE1)) we uncovered that there are some inconsistencies in our error messages for the Name field on A, AAAA, MX, CNAME, and TXT records. These are the issues:
* Issue 1:  A Name field accepts an @ symbol when it is the only character used. If it is used with other characters, it is invalid and will throw an error. However, the error message we display in this case is, "Enter the name without using parentheses, colons, or semicolons."  
* Issue 2: The Name field does not accept consecutive dots (...) or leading or trailing dots. However, the error message we display in this case is, "Enter the name without using consecutive periods."

### Acceptance criteria

- [x] Update the Alerts messages spreadsheet with revised messages to address Issue 1 & 2
- [x] Create Implementation ticket to make changes as shown in the [alert messaging spreadsheet](https://docs.google.com/spreadsheets/d/1hF7_n_zo3EpfbgqqqPUMtSMZXynMtRMsBMlcIqZ91ns/edit?gid=987899751#gid=987899751)

### Additional context

Review the [canvas ](https://dhscisa.enterprise.slack.com/docs/T02QH7E1MHA/F097U65KVE1)in the dns hosting review channel for more detail

**Comment by montecristo99** [comment]
Updated the [alert spreadsheet](https://docs.google.com/spreadsheets/d/1hF7_n_zo3EpfbgqqqPUMtSMZXynMtRMsBMlcIqZ91ns/edit?gid=987899751#gid=987899751) with revised messaging.

**Comment by montecristo99** [comment]
Created implementation ticket:
- #5221
